python_modules/dagster/README.md

```diff
@@ -26,55 +16,59 @@
   <a target="_blank" href="https://pypi.org/project/dagster/" style="background:none">
     <img src="https://img.shields.io/pypi/pyversions/dagster?labelColor=4F43DD&color=163B36">
   </a>
-</p>
-
-# Dagster
+  <a target="_blank" href="https://twitter.com/dagster" style="background:none">
+    <img src="https://img.shields.io/badge/twitter-dagster-blue.svg?labelColor=4F43DD&color=163B36&logo=twitter" />
+  </a>
+  <a target="_blank" href="https://dagster.io/slack" style="background:none">
+    <img src="https://img.shields.io/badge/slack-dagster-blue.svg?labelColor=4F43DD&color=163B36&logo=slack" />
+  </a>
+  <a target="_blank" href="https://linkedin.com/showcase/dagster" style="background:none">
+    <img src="https://img.shields.io/badge/linkedin-dagster-blue.svg?labelColor=4F43DD&color=163B36&logo=linkedin" />
+  </a>
+</div>
 
-Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports.
+__Dagster is a cloud-native orchestrator for the whole development lifecycle, with integrated lineage and observability, a declarative programming model, and best-in-class testability.__
 
-You declare functions that you want to run and the data assets that those functions produce or update. Dagster then helps you run your functions at the right time and keep your assets up-to-date.
+Dagster is a data pipeline orchestrator designed for **developing and maintaining data assets**, such as tables, data sets, machine learning models, and reports.
 
-Dagster is built to be used at every stage of the data development lifecycle - local development, unit tests, integration tests, staging environments, all the way up to production.
+With Dagster, you declare—as Python functions—the data assets that you want to build. Dagster then helps you run your functions at the right time and keep your assets up-to-date.
 
-If you're new to Dagster, we recommend reading about its [core concepts](https://docs.dagster.io/concepts) or learning with the hands-on [tutorial](https://docs.dagster.io/tutorial).
-
-An asset graph defined in Python:
+Here is an example of a graph of three assets defined in Python:
 
 ```python
 from dagster import asset
 from pandas import DataFrame, read_html, get_dummies
 from sklearn.linear_model import LinearRegression
 
-@asset
+@asset(compute_kind="Python")
```

Comment:
IMO we should keep the `compute_kind`s out, because it looks like extra boilerplate

Reply:
I appreciate the value of pairing this right down.  I took out the descriptions for the assets as they added extra lines.  I would favor keeping the `compute_kind` because they do make the asset graph a lot more engaging.

Reply:
For a README I share @sryza's instinct here that we should keep the code example minimal and omit the `compute_kind`

Reply:
OK, changes are reverted to the minimal code and basic asset graph.